fuel-moshi/src/main/kotlin/com/github/kittinunf/fuel/moshi/FuelMoshi.kt

```diff
@@ -18,13 +22,14 @@ inline fun <reified T : Any> Request.responseObject(handler: ResponseHandler<T>)
 inline fun <reified T : Any> Request.responseObject() = response(moshiDeserializerOf(T::class.java))
 
 fun <T : Any> moshiDeserializerOf(clazz: Class<T>) = object : ResponseDeserializable<T> {
-    override fun deserialize(content: String): T? = Moshi.Builder()
-                .build()
-                .adapter(clazz)
-                .fromJson(content)
+    override fun deserialize(content: String): T? = defaultMoshi
+            .build()
+            .adapter(clazz)
+            .fromJson(content)
 }
 
 inline fun <reified T : Any> moshiDeserializerOf(adapter: JsonAdapter<T>) = object : ResponseDeserializable<T> {
-    override fun deserialize(content: String): T? =
-            adapter.fromJson(content)
+    override fun deserialize(content: String): T? = adapter.fromJson(content)
+
+    override fun deserialize(inputStream: InputStream): T? = adapter.fromJson(Okio.buffer(Okio.source(inputStream)))
```

Comment:
Do you really need `Okio` here?

Reply:
One of Moshi's JSON Adapter takes Okio.BufferedSource

Reply:
Got it.

Then lgtm